definitions/artifacts/azure-storage-account-blob.json

```diff
@@ -38,7 +29,43 @@
           }
         },
         "security": {
-          "$ref": "../types/azure-security.json"
+          "type": "object",
+          "title": "Security",
+          "description": "Azure Security Configuration",
+          "additionalProperties": false,
+          "required": ["iam"],
```

Comment:
You probably don't want to make the IAM required. You probably want to say that one of or both of are required using a dependency block.